pandas/tests/reshape/test_pivot.py

```diff
@@ -371,6 +371,40 @@ def test_pivot_periods(self):
         pv = df.pivot(index='p1', columns='p2', values='data1')
         tm.assert_frame_equal(pv, expected)
 
+    def test_pivot_with_list_like_values(self):
+        # issue #17160
+        df = pd.DataFrame({'foo': ['one', 'one', 'one', 'two', 'two', 'two'],
+                           'bar': ['A', 'B', 'C', 'A', 'B', 'C'],
+                           'baz': [1, 2, 3, 4, 5, 6],
+                           'zoo': ['x', 'y', 'z', 'q', 'w', 't']})
+
+        result_list = df.pivot(index='zoo', columns='foo',
+                               values=['bar', 'baz'])
+        result_array = df.pivot(index='zoo', columns='foo',
+                                values=np.array(['bar', 'baz']))
+        result_series = df.pivot(index='zoo', columns='foo',
+                                 values=pd.Series(['bar', 'baz']))
+        result_index = df.pivot(index='zoo', columns='foo',
+                                values=pd.Index(['bar', 'baz']))
+
+        data = [[np.nan, 'A', np.nan, 4],
+                [np.nan, 'C', np.nan, 6],
+                [np.nan, 'B', np.nan, 5],
+                ['A', np.nan, 1, np.nan],
+                ['B', np.nan, 2, np.nan],
+                ['C', np.nan, 3, np.nan]]
+        index = Index(data=['q', 't', 'w', 'x', 'y', 'z'], name='zoo')
+        columns = MultiIndex(levels=[['bar', 'baz'], ['one', 'two']],
```

Comment:
add a test with a MultiIndex and pass values as a tuple

Reply:
Are you looking for something like the following? (please correct me if I am wrong)
```
    bar          baz       
  first second first second
0   one      A     1      x
1   one      B     2      y
2   one      C     3      z
3   two      A     4      q
4   two      B     5      w
5   two      C     6      t 
```

then pivot it:
``` 
df.pivot(index=('bar', 'first'), columns=('bar', 'second'), values=('baz', 'first'))
 ```

so the output would be:
```
     A   B   C
one  1   2   3
two  4   5   6
```

Reply:
ping @jreback

Reply:
yep